utils/audio.js

```diff
@@ -1,18 +1,101 @@
 'use strict';
 
-module.exports = {
-  buildSfxCollection(game, componentName, clipCount) {
-    // FIXME: Make this a class.
+class Audio {
+  constructor(game) {
+    this.game = game;
+  }
+
+  buildSfxCollection(componentName, clipCount) {
```

Comment:
Changing the signature here will probably break the game, I think, since you haven't changed the calls to it.